flink-connectors/flink-sql-connector-kinesis/pom.xml

```diff
@@ -35,17 +35,6 @@ under the License.
 
 	<packaging>jar</packaging>
 
-	<dependencyManagement>
-		<dependencies>
-			<dependency>
-				<!-- Bumped for security purposes to a common version in the Hadoop ecosystem -->
-				<groupId>com.google.guava</groupId>
-				<artifactId>guava</artifactId>
-				<version>27.0-jre</version>
-			</dependency>
-		</dependencies>
-	</dependencyManagement>
```

Comment:
the `flink-connector-kinesis` dependency (which is a part of the flink-connectors project) further below already declares Guava in its `pom.xml`. So we can get Guava transitively through `flink-connector-kinesis` and manage the Guava dependency in `flink-connector-kinesis/pom.xml` in a "one source of truth" way

Reply:
Thanks 👍